kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -370,61 +404,12 @@ define([
          * @param {object} message
          */
         function handleJobInfo(_, message) {
-            const jobId = message.jobId;
-            jobManager.model.setItem(`exec.jobs.params.${jobId}`, message.jobInfo.job_params[0]);
-            jobManager.removeListener(jobId, 'job-info');
-            // update the table
-            dataTable
-                .DataTable()
-                .row('#job_' + jobId)
-                .invalidate()
-                .draw();
-        }
-
-        /**
-         * @param {object} _ - job manager context
-         * @param {object} message
-         * @returns
-         */
-        function handleJobDoesNotExist(_, message) {
             const { jobId } = message;
-            jobManager.removeListener(message.jobId, 'job-info');
-            handleJobStatus({
-                jobId,
-                jobState: {
-                    job_id: jobId,
-                    status: 'does_not_exist',
-                },
-            });
-        }
-
-        /**
-         * Update the job status table with the new job status
-         * @param {object} _ - job manager context
-         * @param {object} message
-         */
-        function handleJobStatus(_, message) {
-            const jobState = message.jobState;
-            const jobId = jobState.job_id;
-            const status = jobState.status;
-
-            jobManager.removeListener(jobId, 'job-does-not-exist');
-
-            if (Jobs.isTerminalStatus(status)) {
-                jobManager.removeListener(jobId, 'job-status');
-                if (status === 'does_not_exist') {
-                    jobManager.removeJobListeners(jobId);
-                }
-            }
+            const jobState = jobManager.model.getItem(`exec.jobs.byId.${jobId}`);
+            const rowIx = jobState && jobState.retry_parent ? jobState.retry_parent : jobId;
 
-            // check if the status has changed; if not, ignore this update
-            const previousStatus = jobManager.model.getItem(`exec.jobs.byId.${jobId}.status`);
-            if (status === previousStatus) {
-                return;
-            }
-
-            // otherwise, update the model using the jobState object
-            jobManager.updateModel([jobState]);
+            // update the table
+            dataTable.DataTable().row(`#job_${rowIx}`).invalidate().draw();
```

Comment:
`.invalidate()` makes sure that any table cell content that comes from supplemental data is regenerated.